windows_app_sdk/simple_window/src/main.rs

```diff
@@ -0,0 +1,113 @@
+#![windows_subsystem = "windows"]
+
+use std::convert::TryFrom;
+
+use bindings::{
+    Microsoft,
+    Microsoft::UI::Xaml::{
+        Application, ApplicationInitializationCallback, Controls::Button, HorizontalAlignment,
+        LaunchActivatedEventArgs, RoutedEventHandler, Window,
+    },
+    Windows::Win32::{
+        Foundation::{BOOL, HWND, RECT},
+        UI::{
+            HiDpi::GetDpiForWindow,
+            WindowsAndMessaging::{
+                GetSystemMetrics, GetWindowRect, SetWindowPos, SM_CXSCREEN, SM_CYSCREEN,
+                SWP_NOMOVE, SWP_NOSIZE,
+            },
+        },
+    },
+};
+
+use windows::{implement, IInspectable, Interface};
+
+#[implement(extend Microsoft::UI::Xaml::Application, override OnLaunched)]
+struct App {
+    _window: Option<Window>,
+}
+
+#[allow(non_snake_case)]
+impl App {
+    fn OnLaunched(&mut self, _: &Option<LaunchActivatedEventArgs>) -> windows::Result<()> {
+        let window = Window::new().unwrap();
+        window.SetTitle("WinUI Desktop, Unpackaged (Rust)")?;
+
+        let button = Button::new()?;
+        button.SetContent(IInspectable::try_from("Click Me")?)?;
+        button.SetHorizontalAlignment(HorizontalAlignment::Center)?;
+        button.Click(RoutedEventHandler::new(|sender, _args| {
+            if let Some(button) = sender {
+                button
+                    .cast::<Button>()?
+                    .SetContent(IInspectable::try_from("Clicked! 🦀")?)?;
+            }
+            Ok(())
+        }))?;
+
+        window.SetContent(&button)?;
+
+        let inspectable = &windows::IInspectable::from(&window);
+        let hwnd = match windows_app::window_handle(inspectable) {
+            Some(hwnd) => HWND(hwnd),
+            _ => panic!("Failed to get native window handle"),
```

Comment:
This can be replaced with `let hwnd = window_handle(...).expect("Failed to bla bla");`